schema/playground/monobehaviour_test.schema

```diff
@@ -0,0 +1,9 @@
+package playground;
+
+type Void {}
+
+component MonoBehaviourTest
```

Comment:
Could have a descriptive name, maybe SpinnerRotation?

Reply:
Renamed component to `SpinnerRotation`.